### PR TITLE
Iss #96 add measurement_type to calibration

### PR DESCRIPTION
**Pull request by stephenholleran**
This follows the issue #96 so it is possible to distinguish a temperature calibration from a humidity calibration for a single thermohygrometer sensor.

Instead of duplicating `measurment_type_id` I made it a definition so it can be referenced from both `measurement_point` and `calibration`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,15 +10,18 @@ Given a version number MAJOR.MINOR.PATCH, increment the:
 Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
 
 ## [0.2.0-2021.XX]
-- Rename `sensor_config` to `logger_measurement_config`
```
Comment (abohara) [inline CHANGELOG.md:13]: @stephenholleran Was this not part of another separate commit ?

**Comment by stephenholleran** [comment]
I just changed ‘Rename’ to ‘Renamed’ to put the changelog into past tense which it seems is the more common tense for version or changelog text.

**Review by kersting — COMMENTED** [review]
It looks good.

**Comment by stephenholleran** [comment]
Hi @abohara and @heikowestermann, would you guys be able to review this pull request? Thanks, Stephen.

**Review by heikowestermann — APPROVED** [review]

**Pull request merged**